### PR TITLE
Add basic instructions on how to use the provider

### PR DESCRIPTION
**Pull request by simo5**

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Some pkcs11-drivers accept/require an initialization string (for example NSS
+softokn), if that is needed add it as follow:
+```
+pkcs11-module-init-args = <initialization string here>
+```
+
+Once the section is properly constructed add the following statement to the
+provider section. If a provider section does not exist make sure to create one
+with all the needed provider (at least the default provider will be needed):
```
Comment (neverpanic) [inline HOWTO.md:44]:
```suggestion
with all the needed providers (at least the default provider will be needed):
```

**Review by neverpanic — APPROVED** [review]
Looks good, minor nitpick.

File: HOWTO.md (hunk 1 above)
Comment (simo5) [reply HOWTO.md]: Fixed

**Comment by beldmit** [comment]
PKCS11_PROVIDER_DEBUG is not documented, LGTM otherwise

**Comment by simo5** [comment]
> 

It is not an installation required variable, it is used for debugging, that's another commit/file/day ... :)

**Pull request merged**